src/Compilers/CSharp/Portable/Syntax/SyntaxKind.cs

```diff
@@ -216,42 +402,72 @@ public enum SyntaxKind : ushort
 
         // keywords with an enum value less than ElifKeyword are considered i.a. contextual keywords
         // additional preprocessor keywords
+        /// <summary>Represents <see langword="elif"/>.</summary>
         ElifKeyword = 8467,
+        /// <summary>Represents <see langword="endif"/>.</summary>
         EndIfKeyword = 8468,
+        /// <summary>Represents <see langword="region"/>.</summary>
         RegionKeyword = 8469,
+        /// <summary>Represents <see langword="endregion"/>.</summary>
         EndRegionKeyword = 8470,
+        /// <summary>Represents <see langword="define"/>.</summary>
         DefineKeyword = 8471,
+        /// <summary>Represents <see langword="undef"/>.</summary>
         UndefKeyword = 8472,
+        /// <summary>Represents <see langword="warning"/>.</summary>
         WarningKeyword = 8473,
+        /// <summary>Represents <see langword="error"/>.</summary>
         ErrorKeyword = 8474,
+        /// <summary>Represents <see langword="line"/>.</summary>
         LineKeyword = 8475,
+        /// <summary>Represents <see langword="pragma"/>.</summary>
         PragmaKeyword = 8476,
+        /// <summary>Represents <see langword="hidden"/>.</summary>
         HiddenKeyword = 8477,
+        /// <summary>Represents <see langword="checksum"/>.</summary>
         ChecksumKeyword = 8478,
+        /// <summary>Represents <see langword="disable"/>.</summary>
         DisableKeyword = 8479,
+        /// <summary>Represents <see langword="restore"/>.</summary>
         RestoreKeyword = 8480,
+        /// <summary>Represents <see langword="r"/>.</summary>
         ReferenceKeyword = 8481,
 
+        /// <summary>Represents <code>$"</code> token.</summary>
```

Comment:
❕ `<code>` is a block element, but we want the inline element `<c>` in these comments.

```suggestion
        /// <summary>Represents <c>$"</c> token.</summary>
```